p2p/docs/config.md

```diff
@@ -0,0 +1,39 @@
+# P2P Config
+
+Here we describe configuration options around the Peer Exchange.
+
+## Seed Mode
+
+`--p2p.seed_mode`
+
+The node operates in seed mode. It will kick incoming peers after sharing some peers.
```

Comment:
What about: The node operates in seed mode. In seed mode, a node continuously crawls the network for peers, and upon incoming connection shares some peers and closes the connection.